scenarios/seir-stockflow/sciml/calibrate.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I think we should hold off on adding `*/sciml/calibrate.json`s to other scenarios. I think the base the original one doesn't work yet and the payloads may have to be a bit different. tl;dr: remove `scenarios/*/sciml/calibrate.json` in this PR?

Reply:
I think the reason that the calibrate fails for the sirhd scenario is a weird XML parsing bug. It fails to parse a part of the AMR for whatever reason. Calibrate works on other models.

Reply:
Specifically, when `MathML.parse_str()` is called on the line `"<ci><mml:msub><mml:mi>infectious</mml:mi><mml:mi>population</mml:mi></mml:msub></ci>"` 

I'm not sure if it's a malformed xml statement or if the parser is just being dumb

Reply:
thanks for opening https://github.com/DARPA-ASKEM/simulation-integration/issues/46!

Reply:
resolving for now since this is in a separate issue